<ui version="4.0" >
 <class>StopWatchBase</class>
 <widget class="QWidget" name="StopWatchBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>StopWatchBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="swFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="2" >
      <widget class="QLabel" name="TextLabel3_2" >
       <property name="text" >
        <string>Lap</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLCDNumber" name="lapLcd" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>20</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="numDigits" >
        <number>11</number>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLCDNumber" name="splitLcd" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>20</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="numDigits" >
        <number>11</number>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="TextLabel4" >
       <property name="text" >
        <string>Split</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="0" >
      <widget class="QFrame" name="lapFrame" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QLCDNumber" name="lapNumLcd" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="numDigits" >
        <number>2</number>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="stopStart" >
       <property name="whatsThis" >
        <string>Starts and stops the stopwatch.</string>
       </property>
       <property name="text" >
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>Resets the stopwatch.</string>
       </property>
       <property name="text" >
        <string>Lap/Split</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
